DEVELOPER_GUIDE.md

```diff
@@ -74,6 +83,74 @@ This is typically created in PyCharm IDE by visiting the `Python Interpreter`, s
 `
 In order to run tests within the PyCharm IDE, ensure the `Python Integrated Tools` / `Testing` / `Default Test Runner` is set to `pytest`.
 
+## Running Workloads
```

Comment:
Nit: `Running Workloads` --> `Setting Up an Local OpenSearch Cluster`

This title should be more descriptive for a couple reasons:
- We already have documentation on the official website that details how to run workloads, so this might confuse users
- Most users seem to be using OSB to benchmark larger and external clusters instead of performance testing their local computer. 

Maybe this section could include a brief explanation why developers might want to setup a local cluster (e.g. for simple feature testing or end to end testing etc.)